### PR TITLE
Runner Updates 

### PR DESCRIPTION
**Pull request by annabaker**
**Description**
Changes to Runner docs (overview and installation):
- New support levels introduced for available platforms (Support and Preview)
- Removed old support level descriptions
- Condensed overlapping/repetitive information to only one section (stuck the information about support levels in the overview instead of the installation section -- IMO it does not seem like it belongs with installation. I did link to it at the top, though)
- Reorganized some things, a few sections seemed out of order 
- Softened the wording in some areas (i.e., removing words like "_only_ " in "only available on the scale plan")

**Reasons**
This is a part of a growing effort to soften the way we message Scale features and supported/preview platforms.

### CODE REVIEW

File: jekyll/_cci2/runner-overview.adoc [hunk 1]
```diff
@@ -32,34 +32,69 @@ There are two key use cases CircleCI aims to meet with the runner:
 
 * Unique compute requirements - Customers who need to run jobs on an environment or architecture that CircleCI does not offer as a resource class can use the runner to fill that need.
 
-== Runner operation
+== Available CircleCI runmer platforms
```
Comment (thomasr8) [inline jekyll/_cci2/runner-overview.adoc:35]:
```suggestion
== Available CircleCI runner platforms
```

File: jekyll/_cci2/runner-overview.adoc [hunk 2]
```diff
@@ -32,34 +32,69 @@ There are two key use cases CircleCI aims to meet with the runner:
...
+have questions around support for your environment and use-case(s). We also invite you to https://circleci.canny.io/cloud-feature-requests[share feedback]
+and contribute to our https://discuss.circleci.com/t/self-hosted-runners-are-here/38159[runner discuss page] to help
+prioritize development efforts from our team!
+
+== Getting Started
+
+To get started with CircleCI runner:
+
+* https://circleci.com/contact/[Contact us] to sign up for a free trial of the Scale plan, if you are not already enrolled
+* Provide your own platform to deploy your CircleCI runners (See the <<Available CircleCI runmer platforms>> section)
```
Comment (thomasr8) [inline jekyll/_cci2/runner-overview.adoc:85]:
```suggestion
* Provide your own platform to deploy your CircleCI runners (See the <<Available CircleCI runner platforms>> section)
```

**Review by thomasr8 — CHANGES_REQUESTED** [review]
Two minor fixes, but look great Anna!

File: jekyll/_cci2/runner-overview.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: The CircleCI runner is currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan or to learn more.
+NOTE: The CircleCI runner is available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan or to learn more.
```
Comment (stalmy) [inline jekyll/_cci2/runner-overview.adoc:12]: 1. your new wording from a similar statement in the other file sounds cleaner: "CircleCI runner is currently available on the..."
2. Does it make sense to push interested readers to reach out if they don't have a sales rep:  "Please reach to your sales representative or contact us <linked to https://circleci.com/contact-us/?cloud> for information on ..."?

File: jekyll/_cci2/runner-installation.adoc [hunk 4]
```diff
@@ -9,105 +9,38 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: CircleCI runner is currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+NOTE: CircleCI runner is currently available on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
```
Comment (stalmy) [inline jekyll/_cci2/runner-installation.adoc]: same comment from the other change:

Does it make sense to push interested readers to reach out if they don't have a sales rep: "Please reach to your sales representative or contact us <linked to https://circleci.com/contact-us/?cloud> for information on ..."?

**Comment by stalmy** [comment]
A couple minor comments/suggestions.

Looks fantastic overall!

**Review by thomasr8 — APPROVED** [review]

File: jekyll/_cci2/runner-installation.adoc [hunk 5]
```diff
@@ -9,97 +9,38 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: CircleCI runner is currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+NOTE: CircleCI runner is available on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative (or https://circleci.com/contact-us/?cloud[contact us]) for information on how to sign up for the Scale Plan.
 
-This page describes how to install CircleCI runner on Linux and macOS platforms.
+This page describes how to install CircleCI runner on Linux and macOS platforms. For other platorms, see xref:runner-overview.adoc#available-circleci-runmer-platforms[Available CircleCI runner platforms] for more information.
```
Comment (teesloane) [inline jekyll/_cci2/runner-installation.adoc:14]:
```suggestion
This page describes how to install CircleCI runner on Linux and macOS platforms. For other platorms, see xref:runner-overview.adoc#available-circleci-runner-platforms[Available CircleCI runner platforms] for more information.
```
I believe that ☝️  should fix your failing build :)

**Pull request merged**